server/src/test/java/org/elasticsearch/search/aggregations/metrics/ScriptedMetricIT.java

```diff
@@ -322,11 +325,11 @@ public void testMap() {
         assertThat(numShardsRun, greaterThan(0));
     }
 
-    public void testMapWithParams() {
+    public void testExplicitAggParam() {
```

Comment:
testMapWithParams was actually working because the params were on the aggregation, not because the params were on the map script. This test may be extraneous now but I renamed it instead of deleting it since it does provide specific coverage for an explicit _agg param, even if that may be kind of a weird case. I wouldn't presume to delete it without an opinion from a dev more familiar with this code..